### DISCUSSION TITLE
GPS coordinates accuracy with Shiny.Locations 3.2.3 and background service on Android

### DISCUSSION THREAD

**Discussion by lpujol31** [shinyorg/shiny#1384] [Q&A]
Hi, I met some problems with Shiny.Locations package when using in a background service
Context : MAUI / .net 8 / Visual Studio 2022 / Android 14.0 API 34
When the app is in the foreground  : everything is fine 
When the app is in the background : it behaves as attempted (I can catch readings with OnGpsReading) but in this case, the locations are not accurate.... (nearly 50/100 meters from the real position)

Is it a know issue ? What am I doing wrong ? 
Thanks for your help, don't hesitate if some additional informations are needed

I called the listener with these parameters :
public const GpsBackgroundMode BackgroundMode = GpsBackgroundMode.Realtime; 
public const GpsAccuracy Accuracy = GpsAccuracy.Highest;

await gpsManager.StartListener(new GpsRequest 
{
    BackgroundMode = Constants.BackgroundMode,
    Accuracy = Constants.Accuracy
});

**Comment by aritchie** [comment]
IF you're getting coordinates, there isn't an issue here I can fix.  This is just how the OS behaves.
